docker-compose.yml

```diff
@@ -125,21 +125,20 @@ services:
     networks:
       - celestia-zkevm-net
 
-  # TODO: Determine why this does not work but running a relayer binary locally built from main
-  # can relay a successful warp transfer.
-  # relayer:
-  #   image: gcr.io/abacus-labs-dev/hyperlane-agent:agents-v1.4.0
-  #   container_name: relayer
-  #   environment:
-  #     - CONFIG_FILES=/app/config/config.json
-  #   volumes:
-  #     - ./hyperlane/relayer-config.json:/app/config/config.json
-  #   command: "/app/relayer"
-  #   depends_on:
-  #     hyperlane-init:
-  #       condition: service_completed_successfully
-  #   networks:
-  #     - celestia-zkevm-net
+  relayer:
+    image: gcr.io/abacus-labs-dev/hyperlane-agent:main
+    platform: linux/amd64
```

Comment:
I tested this by deleting my local image and restarting the compose services and letting it pull:

```
docker image rm {image_id}
docker compose up
```